### ISSUE TITLE
Mer Bleue Event Workback Plan

### ISSUE THREAD

**Issue by MartinNesbitt** [ForetCapitaleForest/actions#392]
### Planning
| Timing | Activity &#9744; &#9745;| Owner | Status |
| --- | --- | --- | --- |
| 4 months out | &#9744;  Begin landowner conversations | |
###Fundraising and Planning
- [ ] Project Budget @MartinNesbitt 
- [ ] Project fundraising. 
    - #437 
- [ ] 

### Site Preparation
- [ ] Initial site selection.
   - initial site selection complete. Must confirm with NCC on the location of the 1st planting.

### Materials and Equipment
- [ ] Trees (species/numbers) @smlambert to check her notes on what the species might before this site as some we may not sourced before.
- [ ] Compost / mulch
- [ ] Tree mats / guards
- [ ] Signage
- [ ] Hospitality
- [ ] Porta-potty
- [ ] Covid related materials

### Volunteer Coordination
- [ ] Identify volunteer needs and sources
- [ ] identify volunteer groups/stewards. Looking to local CAs or maybe Mer Bleue Outdoor clubs?

### Promotion
- [ ] Finalize and approve signage

### Event Day Logistics
- [ ] Weather forecast (call it)

### Evaluation
- [ ] Post-event email

### Planting with a Plan
- [ ] Site photography
- [ ] Monitoring and Maintenance Plan

**Comment by smlambert** [comment]
When we first did the walk-about with Cedric, Bruno and Andy, we talked about species that can stand wet soil and occasional flooding (given this planting site is along a creek).  If I recall correctly, Balsam poplar, silver maple and willow were part of the list.  We can also check with Andy and NCC.  I purchased a lot of silver maple this spring intentionally as I thought we would want/need it for this site.
